src/serialize.h

```diff
@@ -23,6 +23,7 @@
 
 #include <prevector.h>
 #include <span.h>
+#include <util/types.h>
```

Comment:
I guess this can be dropped, though I see how it might be used in this file for `SERIALIZE_METHODS`.